setup.cfg

```diff
@@ -38,7 +38,7 @@ setup_requires =
     sphinx_rtd_theme
 install_requires =
     numpy
-    scipy != 1.4.0
+    scipy > 1.4.0, < 1.6
```

Comment:
What happens with scipy 1.6?

Reply:
sklearn < 0.24 breaks with scipy > 1.6
But once we move to sklearn == 0.24, this can be removed.

Reply:
added a comment TODO